docs/unified-test-documentation/dasharo-security/206-secure-boot.md

```diff
@@ -499,3 +499,914 @@ incorrect format.
+
+**Test steps**
+
+1. Power on the DUT.
+1. While the DUT is booting, hold the `BIOS_SETUP_KEY` to enter the UEFI setup
+    menu.
+1. Enter the `Device Manager` menu using the arrow keys and Enter.
+1. Enter the `Secure Boot Configuration` submenu.
+1. Set the `Secure Boot Mode` field to `Custom Mode`.
+1. Erase Secure Boot keys select options in the given order: `Custom Secure Boot Options` ->
+    `DB Options` -> `Enroll Signature` -> `Erase all Secure Boot Keys`
+1. Press `F10` to save changes.
+1. Press `ESC` until the setup menu.
+1. Select the `Reset` option.
+1. The DUT will now attempt to boot `OPERATING_SYSTEM`.
+1. Login to `OPERATING_SYSTEM`.
+1. Remove Old Secure Boot keys:
+
+    ```bash
+    rm -rf /usr/share/secureboot
+    ```
+
+    > Note: `root` right might be needed.
+
+1. Generate new Secure Boot keys:
+
+    ```bash
+    sbctl create-keys
+    ```
+
+    > Note: `root` rights might be needed.
+
+1. Enroll generated Secure Boot keys:
+
+    ```bash
+    sbctl enroll-keys --yes-this-might-brick-my-machine
+    ```
+
+    > Note: `root` rights might be needed.
+
+1. Sign all components in `OPERATING_SYSTEM`:
+
+    ```bash
+    sbctl verify | awk -F ' ' '{print $2}' | tail -n+2 | xargs -I "#" sbctl sign "#"
+    ```
+
+    > Note: `root` rights might be needed.
+
+1. Reboot `OPERATING_SYSTEM`.
+1. While the DUT is booting, hold the `BIOS_SETUP_KEY` to enter the UEFI setup
+    menu.
+1. Enter the `Device Manager` menu using the arrow keys and Enter.
+1. Enter the `Secure Boot Configuration` submenu.
+1. Set the `Current Secure Boot State` field to `Enabled`.
+1. Press `F10` to save changes.
+1. Press `ESC` until the setup menu.
+1. Select the `Reset` option.
+1. The DUT will now attempt to boot `OPERATING_SYSTEM`.
+1. Login to `OPERATING_SYSTEM`.
+1. Check if Secure Boot is enabled:
+
+    ```bash
+    dmesg | grep secureboot
+    ```
+
+    > Note: `root` rights might be needed.
+
+**Expected result**
+
+1. In `dmesg` output should be a line informing that Secure Boot is enabled.
+
+## SBO013.001 Check automatic certificate provisioning
+
+**Test description**
+
+This test verifies that the automatic certificate provisioning will install
+custom keys which will make Ubuntu unbootable. Before launching test, make sure
+that DTS with automatic certificate provisioning is attached and can be booted
+on DUT.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo.
+1. `OPERATING_SYSTEM` = Ubuntu 22.04.
+1. Additional `USB storage`for keeping Dasharo Tools Suite.
+1. Dasharo Tools Suite with UEFI secure boot support.
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../generic-test-setup.md#firmware).
+1. Proceed with the
+    [Generic test setup: OS installer](../generic-test-setup.md#os-installer).
+1. Proceed with the
+    [Generic test setup: OS installation](../generic-test-setup.md#os-installation).
+1. Proceed with the [DTS: Build image with UEFI Secure Boot
```

Comment:
@TomaszAIR I suppose DTS has functionality of automatic certificate provisioning,  on current release, hasn't it?

Reply:
@DaniilKl the DTS image that we are releasing https://github.com/Dasharo/meta-dts/releases does not have UEFI SB integrated, if we want image with it we need to follow linked instruction.